src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+
+	protected boolean isPrivate;
+	public String value;
```

Comment:
Is `public` the best access modifier for `value` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.